core/commands/keystore.go

```diff
@@ -143,6 +151,159 @@ func formatID(id peer.ID, formatLabel string) string {
 	}
 }
 
+var keyExportCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Export a keypair",
+		ShortDescription: `
+Exports a named libp2p key to disk.
+
+By default, the output will be stored at './<key-name>', but an alternate
```

Comment:
Maybe add a `.key` extension?

Reply:
Sure, that seems reasonable. I'll add it now.

Reply:
Done